src/lambdas/get-location.ts

```diff
@@ -0,0 +1,46 @@
+import fetch from 'node-fetch'
+
+/*
+Return the coordinates of a given address using the MapBox Places API
+
+eg. provide an address to query = '1600 Pennsylvania Ave NW, Washington, DC'
```

Comment:
I tried to test this address using Insomnia, but I am getting `null`. Do you know the reason?
![image](https://user-images.githubusercontent.com/46347630/148130436-bcd4d78a-bba5-4a67-acbb-3dc65e796ba1.png)

Reply:
Apparently it was a line break when construction the mapbox url. Should be fixed now.

Reply:
it is working now :slightly_smiling_face: